src/components/FindMovie/FindMovie.tsx

```diff
@@ -20,38 +73,84 @@ export const FindMovie: React.FC = () => {
             />
           </div>
 
-          <p className="help is-danger" data-cy="errorMessage">
-            Can&apos;t find a movie with such a title
-          </p>
+          {error && (
+            <p className="help is-danger" data-cy="errorMessage">
+              Can&apos;t find a movie with such a title
+            </p>
+          )}
         </div>
 
         <div className="field is-grouped">
           <div className="control">
-            <button
-              data-cy="searchButton"
-              type="submit"
-              className="button is-light"
-            >
-              Find a movie
-            </button>
-          </div>
+            {movie ? (
+              <button
+                data-cy="searchButton"
+                type="submit"
+                className={classNames(
+                  'button',
+                  'is-ligth',
+                  {
+                    'is-loading': spinner,
+                  },
+                )}
+                disabled={query.length === 0}
+                onClick={() => setSpinner(true)}
+              >
+                Search again
+              </button>
+            ) : (
+              <button
+                data-cy="searchButton"
+                type="submit"
+                className={classNames(
+                  'button',
+                  'is-ligth',
+                  {
+                    'is-loading': spinner,
+                  },
+                )}
+                disabled={query.length === 0}
+                onClick={() => setSpinner(true)}
+              >
+                Find a movie
+              </button>
+            )}
```

Comment:
These are the same buttons, only text changes
```suggestion
            <button
                data-cy="searchButton"
                type="submit"
                className={classNames(
                  'button',
                  'is-ligth',
                  {
                    'is-loading': spinner,
                  },
                )}
                disabled={query.length === 0}
                onClick={() => setSpinner(true)}
              >
                {movie ? 'Search again' : 'Find a movie}
              </button>
```